test/compilable/test9701.d

```diff
@@ -0,0 +1,33 @@
+// https://issues.dlang.org/show_bug.cgi?id=9701
+
+template AliasSeq(TList...)
+{
+    alias AliasSeq = TList;
+}
+
+ @disable enum Enum
+{
+    value0,
+    @("uda1") value1,
+    @("uda2", "uda3", 42) value2,
+    @disable disabled,
+    deprecated deprecated_
+}
+
+@("uda0")
+enum
+{
+    value0,
+    @("uda1") value1,
+    @("uda2") @("uda3") value2,
```

Comment:
Should be a test with the `@uda` syntax as well.

Reply:
Done